src/components/Card/Card.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import {activeOpacity, metrics} from '../../helpers/metrics'
+import type {ITheme} from '../../theme'
+import type {ViewProps} from 'react-native'
+
+export type CardProps = {
+  onPress?: () => void
+  children?: React.ReactNode
+  bgColor?: string
+  style?: ViewProps
+  testId?: string
+}
+
+const Card: React.FC<CardProps> = ({onPress, children, bgColor, style, testId}) => (
+  <View
+    onPress={onPress}
+    activeOpacity={onPress ? activeOpacity.low : activeOpacity.none}
+    bgColor={bgColor}
```

Comment:
why you need to define separate bgColor props ?

Reply:
ok. I removed it